atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -175,15 +178,15 @@ public int execute(final AtlasDbServices services) {
                         ? sweepRunner.dryRun(tableToSweep, batchConfig, startRow.get())
                         : sweepRunner.run(tableToSweep, batchConfig, startRow.get());
                 printer.info(
-                        "Swept from {} to {} in table {} in {} ms, examined {} cell values,"
-                                + " {}deleted {} stale versions of those cells.",
```

Comment:
maybe a bit repetitive, but should we keep both the [DRY RUN] flag and this? Main concern is whether as an engineer I might panic if I come across this in the field (before realising that it's a dry run).